deploy/complete/helm-chart/mushop/charts/carts/templates/oadb-secrets.yaml

```diff
@@ -1,7 +1,7 @@
 {{- $usesOsb := (index (.Values.global | default .Values) "osb").atp | default .Values.osb.atp -}}
 {{- $oadbService := .Values.secrets.oadbService | default .Values.global.oadbService -}}
 {{- $oadbWalletPw := .Values.secrets.oadbWalletPassword | default .Values.global.oadbWalletPassword -}}
-{{- $oadbUser := .Values.secrets.oadbUser | default (printf "%s_user" .Chart.Name) -}}
+{{- $oadbUser := .Values.secrets.oadbUser | default (printf "%s_%s_user" .Release.Namespace .Chart.Name ) | trunc 96 -}}
```

Comment:
truncate to 96?

Reply:
yes, to account for `_ROLE` as well (plus quite a few spares)

Reply:
Total can be 128. 96 is safe.